proto/axelar/auxiliary/v1beta1/tx.proto

```diff
@@ -15,10 +15,7 @@ message BatchRequest {
   option (permission.exported.v1beta1.permission_role) = ROLE_UNRESTRICTED;
   option (cosmos.msg.v1.signer) = "sender";
 
-  bytes sender_bz = 1 [
-    (gogoproto.casttype) = "github.com/cosmos/cosmos-sdk/types.AccAddress",
-    deprecated = true
-  ];
+  reserved 1; // used to be sender_bz
```

Comment:
this field used the be name `sender` with type sdk.AccAddress

Reply:
I updated this for all the messages where it's the case (hope I didn't miss any)